demo/src/screens/componentScreens/FeatureHighlightScreen.js

```diff
@@ -61,29 +60,53 @@ class FeatureHighlightScreen extends Component {
     }
   }
 
-  moveNext() {
+  moveNext = () => {
     const {currentTargetIndex} = this.state;
     const newTargetIndex = currentTargetIndex + 1;
 
-    if (newTargetIndex < _.size(this.targets)) {
-      this.setState({currentTargetIndex: newTargetIndex});
+    this.moveToPage(newTargetIndex);
+  }
+
+  moveToPage(index) {
+    if (index < _.size(this.targets)) {
+      this.setState({currentTargetIndex: index});
     } else {
       this.closeHighlight();
     }
   }
 
+  getPageControlProps() {
+    return {
+      numOfPages: titles.length, 
+      currentPage: this.state.currentTargetIndex, 
+      onPagePress: this.onPagePress,
+      color: Colors.dark30,
+      inactiveColor: Colors.dark80,
+      size: 8
+    };
+  }
+
+  onPagePress = (index) => {
+    this.moveToPage(index);
+  }
+
   renderHighlighterOverlay() {
     const {showFTE, currentTargetIndex} = this.state;
+    const lastPage = titles.length - 1;
+
     return (
       <FeatureHighlight
         visible={showFTE}
         title={titles[currentTargetIndex]}
         message={messages[currentTargetIndex]}
+        headline={currentTargetIndex === lastPage ? headlines[0] : undefined}
         confirmButtonProps={{label: 'Got It', onPress: this.moveNext}}
         onBackgroundPress={this.closeHighlight}
         getTarget={() => this.targets[currentTargetIndex]}
         // highlightFrame={{x: 30, y: 70, width: 150, height: 30}}
         // highlightFrame={{x: 160, y: 336, width: 150, height: 56}}
+        // borderRadius={4}
```

Comment:
you can set for the example the first item to be round and the reset be square so users can see it's customizable

Reply:
I know. I didn't want to change the example. Done.